dnsmenace.py

```
@@ -1144,6 +1469,572 @@ async def bulk_lookup():
+    Query DNS over HTTPS (DoH) - bypasses local DNS filtering.
+
+    Uses encrypted HTTPS connections to DNS servers, providing privacy
+    and bypassing local network DNS interception.
+
+    Example:
+        dnsmenace doh google.com
+        dnsmenace doh example.com -t A -t AAAA -p cloudflare
+    """
+    print_banner()
+
+    console.print(f"\n[bold cyan]DNS over HTTPS Query[/bold cyan]")
```

Ruff (0.14.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)